setup.py

```diff
@@ -19,7 +19,7 @@
 deps = ['mozinfo >= 0.7',
         'mozlog >= 1.3',
         'progressbar == 2.2',
-        'requests == 1.2.2',
+        'requests >= 2.7.0',
```

Comment:
For packages outside of our control we do not want this non-strict version matches. Please keep the double equal signs as what we had before. Thanks. Once you update the PR please also add a comment. I will land this patch then.